data/json/professions.json

```
@@ -3720,17 +3726,33 @@
   {
     "type": "profession",
     "id": "priest",
-    "name": "Priest",
-    "description": "Armageddon has come!  You did everything you could to protect your parish faithful, but it appears that prayers were not enough.  Now that they are all dead, you should probably find something more tangible to protect you.",
+    "name": { "male": "Protestant Priest", "female": "Churchwoman" },
```

```suggestion
    "name": { "Pastor },
```
Male and female are both pastors in protestant traditions except the ones that don't allow women but those are less common in NE

---

Didn't know, thanks